.travis.yml

```diff
@@ -21,7 +21,7 @@ before_deploy:
 - ./.travis/install_install4j
 - ENGINE_VERSION=$(grep "engine_version" game_engine.properties | sed 's/.*= *//g')
 - TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
-- ./.travis/build_installers $TAGGED_VERSION
+- TAGGED_VERSION=$TAGGED_VERSION ./gradlew release
```

Comment:
I don't quite follow here in this assignment to `TAGGED-VERSION`. Could you the clarify what is being done?